src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTestForInvalidFiles.java

```diff
@@ -258,6 +258,22 @@ public void should_not_parse_envelope_with_duplicate_payment_dcn_numbers() throw
             .hasMessageContaining("instance: {\"pointer\":\"/payments\"}");
     }
 
+    @Test
+    public void should_not_parse_envelope_with_invalid_zip_file_name_in_rescan_for_property() throws IOException {
```

Comment:
will metadata with entry like `"rescan_for" : null` pass validation?

Reply:
let's add an extra test to verify it passes validation, when field is not included (similar to `should_parse_envelope_data_with_no_case_number`)